env/dev/sim/spack.yaml

```diff
@@ -19,6 +19,10 @@ spack:
   concretization: together
   view: false
   packages:
+    all:
+      variants: cxxstd=17
+    xrootd:
+      variants: cxxstd=11
```

Comment:
or try xrootd 5?

Reply:
Will do that in the bump versions branch.

Also my intention is to bump spack very soon, which might help in this area as well (xrootd recipe is strange).